script/01_DeployVault.s.sol

```diff
@@ -13,12 +15,30 @@ import {Vault} from "../src/Vault.sol";
  *     --verify
  */
```

Comment:
possible to also update the comments here? as `--verify` will no longer work. eg.

```
/**
 * Step 1:
 * forge script script/01_DeployVault.s.sol:DeployVaultScript -vvv \
 *     --rpc-url $RPC_URL \
 *     --broadcast \
 *     --slow \
 *     --verify
 *
 * Step 2:
 * forge verify-contract <address> Vault --watch --chain <chain_id>
 */
```

Reply:
On it